src/app/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,50 @@
+/* Import common styling */
+@import "../layout/variables";
+
+/* Sidebar container */
+.side-nav-content {
+    color: $white;
+    width: 200px;
+    margin: 10px;
+
+    // ul list of the sidebar
+    .nav-list {
+        list-style: none;
+        height: 100%;
+
+        // Sidebar li list
+        .nav-list-item {
+            display: flex;
+            align-items: center;
+            height: 50px;
+
+            // Icons of sidebar
+            i {
+                font-size: 20px;
+                height: 25px;
+                width: 25px;
+                margin: 0 10px 0 10px;
+                text-align: center;
+                line-height: 25px;
+            }
+
+            // Anchor tag for sidebar
+            a {
+                text-decoration: none;
+                color: $white;
+            }
+
+            // Sidebar list hover
+            &:hover {
+                background-color: $light-grey;
+                border-radius: 10px;
+                color: $green;
+
+                // Sidebar anchor tag color change
+                a {
+                    color: $green;
+                }
+            }
+        }
+    }
+}
```

Comment:
More than 4 level hierarchy is not allowed.

Reply:
Di done changes